tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -497,11 +505,15 @@ impl<'a> TestRunner<'a> {
             prepare_package(self.file_manager, self.parsed_files, package);
         check_crate_and_report_errors(&mut context, crate_id, &self.args.compile_options)?;
 
-        Ok(context
-            .get_all_test_functions_in_crate_matching(&crate_id, &self.pattern)
-            .into_iter()
-            .map(|(test_name, _)| test_name)
-            .collect())
+        let mut tests = context.get_all_test_functions_in_crate_matching(&crate_id, &self.pattern);
+        if self.args.no_fuzz {
+            tests.retain(|(_, test)| !test.has_arguments());
+        }
+        if self.args.only_fuzz {
+            tests.retain(|(_, test)| test.has_arguments());
```

Comment:
What do you think of marking these tests as skipped, e.g. `test_name -- SKIP`?

Reply:
Oh! I thought about doing it that way, then... I forgot 😅 I think that makes sense, I'll do the change.

Reply:
Done!